edenia/eden-spend-explorer
# Feat/create views and query to expenses graphs 154

### Create views and query to expenses graphs

### What does this PR do?

- Resolve #154 

### Steps to test

1. Run project
2. Wait to load data
3. Enter to localhost:9695 and see the data for expenses like this:
![imagen](https://user-images.githubusercontent.com/49013556/194435913-794b3809-5f1c-4d83-90d6-fd6cd31eebe7.png)


#### CheckList
